server/src/main/java/org/uiautomation/ios/utils/SimulatorSettings.java

```diff
@@ -56,7 +60,7 @@
   private InstrumentsVersion instrumentsVersion;
 
   public SimulatorSettings(HostInfo info, String sdkVersion, boolean is64bit, DeviceType device,
-                           DeviceVariation variation) {
+                           DeviceVariation variation) throws NotActiveException {
```

Comment:
How about I create a new Exception name `DeviceNotAvailableException` or any name you prefer?